new/detector/implementation/java/ruby_test.go

```diff
@@ -0,0 +1,23 @@
+package ruby_test
```

Comment:
```suggestion
package java_test
```
(also filename and test names)